Print IDEasy version in ide status when offline

### Expected behavior

As an IDEasy user, I expect to get relevant information such as my current IDEasy version, when calling `ide status` even when I am offline.

```
$ ide status
IDE_ROOT is set to D:\projects
IDE_HOME is set to D:\projects\IDEasy
You are offline. Check your internet connection and potential proxy settings.
Your version of IDEasy is 2025.03.001.
```

### Actual behavior
```
$ ide status
Updates are available for the settings repository. If you want to apply the latest changes, call "ide update"
IDE_ROOT is set to D:\projects
IDE_HOME is set to D:\projects\IDEasy
You are offline. Check your internet connection and potential proxy settings.
Your settings are not up-to-date, please run 'ide update'. 
An unexpected error occurred!
We are sorry for the inconvenience.
Please check the error below, resolve it and try again.
If the error is not on your end (network connectivity, lack of permissions, etc.) please file a bug:
https://github.com/devonfw/IDEasy/issues/new?assignees=&labels=bug&projects=&template=bug.md&title=IllegalStateException%3A+Failed+to+fetch+XML+metadata+from+https%3A%2F%2Fs01.oss.sonatype.org%2Fcontent%2Frepositories%2Fsnapshots%2Fcom%2Fdevonfw%2Ftools%2FIDEasy%2Fide-cli%2Fmaven-metadata.xml
java.lang.IllegalStateException: Failed to fetch XML metadata from https://s01.oss.sonatype.org/content/repositories/snapshots/com/devonfw/tools/IDEasy/ide-cli/maven-metadata.xml
        at com.devonfw.tools.ide.tool.repository.MavenRepository.fetchXmlMetadata(MavenRepository.java:307)
        at com.devonfw.tools.ide.tool.repository.MavenRepository.fetchVersions(MavenRepository.java:214)
        at com.devonfw.tools.ide.tool.repository.MavenRepository.resolveVersion(MavenRepository.java:201)
        at com.devonfw.tools.ide.tool.repository.MavenRepository.resolveVersion(MavenRepository.java:186)
        at com.devonfw.tools.ide.tool.IdeasyCommandlet.getLatestVersion(IdeasyCommandlet.java:122)
        at com.devonfw.tools.ide.tool.IdeasyCommandlet.checkIfUpdateIsAvailable(IdeasyCommandlet.java:132)
        at com.devonfw.tools.ide.commandlet.StatusCommandlet.run(StatusCommandlet.java:45)
        at com.devonfw.tools.ide.context.AbstractIdeContext.applyAndRun(AbstractIdeContext.java:958)
        at com.devonfw.tools.ide.context.AbstractIdeContext.run(AbstractIdeContext.java:876)
        at com.devonfw.tools.ide.cli.Ideasy.runOrThrow(Ideasy.java:95)
        at com.devonfw.tools.ide.cli.Ideasy.run(Ideasy.java:59)
        at com.devonfw.tools.ide.cli.Ideasy.main(Ideasy.java:133)
        at java.base@21.0.2/java.lang.invoke.LambdaForm$DMH/sa346b79c.invokeStaticInit(LambdaForm$DMH)
Caused by: java.net.ConnectException: Connection timed out: connect
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.jni.functions.JNIFunctions$NewObjectWithObjectArrayArgFunctionPointer.invoke(JNIFunctions.java)
        at org.graalvm.nativeimage.builder/com.oracle.svm.core.jni.functions.JNIFunctions.ThrowNew(JNIFunctions.java:907)
        at java.base@21.0.2/sun.nio.ch.Net.connect0(Native Method)
        at java.base@21.0.2/sun.nio.ch.Net.connect(Net.java:589)
        at java.base@21.0.2/sun.nio.ch.Net.connect(Net.java:578)
        at java.base@21.0.2/sun.nio.ch.NioSocketImpl.connect(NioSocketImpl.java:583)
        at java.base@21.0.2/java.net.Socket.connect(Socket.java:751)
        at java.base@21.0.2/java.net.Socket.connect(Socket.java:686)
        at java.base@21.0.2/sun.net.NetworkClient.doConnect(NetworkClient.java:183)
        at java.base@21.0.2/sun.net.www.http.HttpClient.openServer(HttpClient.java:531)
        at java.base@21.0.2/sun.net.www.http.HttpClient$1.run(HttpClient.java:590)
        at java.base@21.0.2/sun.net.www.http.HttpClient$1.run(HttpClient.java:588)
        at java.base@21.0.2/java.security.AccessController.executePrivileged(AccessController.java:114)
        at java.base@21.0.2/java.security.AccessController.doPrivileged(AccessController.java:571)
        at java.base@21.0.2/sun.net.www.http.HttpClient.privilegedOpenServer(HttpClient.java:587)
        at java.base@21.0.2/sun.net.www.http.HttpClient.openServer(HttpClient.java:631)
        at java.base@21.0.2/sun.net.www.protocol.https.HttpsClient.<init>(HttpsClient.java:264)
        at java.base@21.0.2/sun.net.www.protocol.https.HttpsClient.New(HttpsClient.java:377)
        at java.base@21.0.2/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.getNewHttpClient(AbstractDelegateHttpsURLConnection.java:202)
        at java.base@21.0.2/sun.net.www.protocol.http.HttpURLConnection.plainConnect0(HttpURLConnection.java:1258)
        at java.base@21.0.2/sun.net.www.protocol.http.HttpURLConnection.plainConnect(HttpURLConnection.java:1123)
        at java.base@21.0.2/sun.net.www.protocol.https.AbstractDelegateHttpsURLConnection.connect(AbstractDelegateHttpsURLConnection.java:179)
        at java.base@21.0.2/sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1675)
        at java.base@21.0.2/sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1599)
        at java.base@21.0.2/sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:223)
        at java.base@21.0.2/java.net.URL.openStream(URL.java:1325)
        at com.devonfw.tools.ide.tool.repository.MavenRepository.fetchXmlMetadata(MavenRepository.java:303)
        ... 12 more
```

 
Error: IDEasy failed with exit code 255

### Steps to reproduce (bug) / Use Case of feature request (enhancement)

1. Disconnect from internet
2. Execute `ide status`

### Related/Dependent Issues
#1110 

### Comments/Hints:


### Affected version:

* OS: Windows/Linux/Mac (most likely all of them, but tested only on Windows so far)

--- a/CHANGELOG.adoc
+++ b/CHANGELOG.adoc
@@ -2,6 +2,14 @@
 
 This file documents all notable changes to https://github.com/devonfw/IDEasy[IDEasy].
 
+== 2025.06.002
+
+Release with new features and bugfixes:
+
+* https://github.com/devonfw/IDEasy/issues/1160[#1160]: Print IDEasy version in ide status when offline
+
+The full list of changes for this release can be found in https://github.com/devonfw/IDEasy/milestone/29?closed=1[milestone 2025.06.002].
+
 == 2025.06.001
 
 Release with new features and bugfixes:

--- a/cli/src/main/java/com/devonfw/tools/ide/commandlet/StatusCommandlet.java
+++ b/cli/src/main/java/com/devonfw/tools/ide/commandlet/StatusCommandlet.java
@@ -54,10 +54,6 @@ public class StatusCommandlet extends Commandlet {
   }
 
   private void checkForUpdate() {
-    if (!this.context.isOnline()) {
-      this.context.warning("Check for newer version of IDEasy is skipped due to no network connectivity.");
-      return;
-    }
     new IdeasyCommandlet(this.context, null).checkIfUpdateIsAvailable();
     logSystemInfo();
   }

--- a/cli/src/main/java/com/devonfw/tools/ide/tool/IdeasyCommandlet.java
+++ b/cli/src/main/java/com/devonfw/tools/ide/tool/IdeasyCommandlet.java
@@ -133,6 +133,11 @@ public class IdeasyCommandlet extends MvnBasedLocalToolCommandlet {
    */
   public boolean checkIfUpdateIsAvailable() {
     VersionIdentifier installedVersion = getInstalledVersion();
+    if (!this.context.isOnline()) {
+      this.context.success("Your version of IDEasy is {}.", installedVersion);
+      this.context.warning("Skipping check for newer version of IDEasy due to lack of network connectivity.");
+      return false;
+    }
     VersionIdentifier latestVersion = getLatestVersion();
     if (installedVersion.equals(latestVersion)) {
       this.context.success("Your version of IDEasy is {} which is the latest released version.", installedVersion);

--- a/cli/src/test/java/com/devonfw/tools/ide/commandlet/StatusCommandletTest.java
+++ b/cli/src/test/java/com/devonfw/tools/ide/commandlet/StatusCommandletTest.java
@@ -40,6 +40,6 @@ public class StatusCommandletTest extends AbstractIdeContextTest {
     status.run();
 
     // assert
-    assertThat(context).logAtWarning().hasMessage("Check for newer version of IDEasy is skipped due to no network connectivity.");
+    assertThat(context).logAtWarning().hasMessage("Skipping check for newer version of IDEasy due to lack of network connectivity.");
   }
 }

